app/assets/stylesheets/layout/_top_menu_mobile.sass

```diff
@@ -108,7 +109,7 @@
         // Viewport minus #account-nav-right when logged out, which is
         // 165px, as this is the space for the long login text label: "Connexion" (French).
         // 36px is the padding around the project name.
-        max-width: calc(100vW - 165px - 36px - #{$hamburger-width + $hamburger-right})
+        max-width: calc(100vw - 160px - 22px)
```

Comment:
Please do not change numbers without updating the comment (when there is an explicit one explaining them). Where is the 22 coming from?
Unfortunately this does not work.  When the projects title is long there is an overlapping. 
<img width="209" alt="bildschirmfoto 2018-06-29 um 15 01 34" src="https://user-images.githubusercontent.com/7457313/42093739-b89f3a8c-7bad-11e8-856d-3e2b738eb5ca.png">